### ISSUE TITLE
py.test fails on various imports

### ISSUE THREAD

**Issue by cjacoby** [ejhumphrey/minst-dataset#1]
I haven't put significant effort into debugging this thing yet. It's possible that a new conda environment might fix the problem. But if we could get this all running on travis it would be easier to tell.

```
minst-dataset git:(master) ✗ py.test -v
========================================================================= test session starts ==========================================================================
platform darwin -- Python 3.5.1, pytest-2.8.5, py-1.4.31, pluggy-0.3.1 -- /Users/cjacoby/miniconda3/envs/ismir2016/bin/python
cachedir: .cache
rootdir: /Users/cjacoby/Documents/devel/minst-dataset, inifile:
collecting 21 items / 3 errorsException ignored in: <bound method AudioFile.__del__ of <claudio.fileio.AudioFile object at 0x107bf19b0>>
Traceback (most recent call last):
  File "/Users/cjacoby/miniconda3/envs/ismir2016/lib/python3.5/site-packages/claudio/fileio.py", line 152, in __del__
    self.close()
  File "/Users/cjacoby/miniconda3/envs/ismir2016/lib/python3.5/site-packages/claudio/fileio.py", line 133, in close
    if self._wave_handle:
AttributeError: 'AudioFile' object has no attribute '_wave_handle'
collected 25 items / 4 errors

minst/tests/test_model.py::test_Observation___init__ PASSED
minst/tests/test_model.py::test_Observation_validate PASSED
minst/tests/test_taxonomy.py::test_InstrumentClassMap___init__ PASSED
minst/tests/test_taxonomy.py::test_InstrumentClassMap_allnames PASSED
minst/tests/test_taxonomy.py::test_InstrumentClassMap_classnames PASSED
minst/tests/test_taxonomy.py::test_InstrumentClassMap_getattr PASSED
minst/tests/test_taxonomy.py::test_InstrumentClassMap_index PASSED
minst/tests/test_taxonomy.py::test_InstrumentClass_indices_match PASSED
minst/tests/test_taxonomy.py::test_normalize_instrument_names PASSED
minst/tests/test_utils.py::test_generate_id::[0] PASSED
minst/tests/test_utils.py::test_generate_id::[1] PASSED
minst/tests/test_utils.py::test_generate_id::[2] PASSED
minst/tests/test_utils.py::test_get_note_distance::[0] PASSED
minst/tests/test_utils.py::test_get_note_distance::[1] PASSED
minst/tests/test_utils.py::test_get_note_distance::[2] PASSED
minst/tests/test_utils.py::test_get_note_distance::[3] PASSED
minst/tests/test_utils.py::test_get_note_distance::[4] PASSED
minst/tests/test_utils.py::test_get_note_distance::[5] PASSED
minst/tests/test_utils.py::test_check_audio_file::[0] PASSED
minst/tests/test_utils.py::test_check_audio_file::[1] PASSED
minst/tests/test_utils.py::test_check_audio_file::[2] PASSED
minst/tests/test_utils.py::test_check_many_audio_files::[0] PASSED
minst/tests/test_utils.py::test_check_many_audio_files::[1] PASSED
minst/tests/test_utils.py::test_check_many_audio_files::[2] PASSED
minst/tests/test_utils.py::test_trim PASSED

================================================================================ ERRORS ================================================================================
______________________________________________________ ERROR collecting minst/tests/test_sources_philharmonia.py _______________________________________________________
minst/tests/test_sources_philharmonia.py:5: in <module>
    import minst.sources.philharmonia as philz
minst/sources/__init__.py:1: in <module>
    import philharmonia as philz
E   ImportError: No module named 'philharmonia'
___________________________________________________________ ERROR collecting minst/tests/test_sources_rwc.py ___________________________________________________________
minst/tests/test_sources_rwc.py:5: in <module>
    import minst.sources.rwc as rwc
minst/sources/__init__.py:1: in <module>
    import philharmonia as philz
E   ImportError: No module named 'philharmonia'
__________________________________________________________ ERROR collecting minst/tests/test_sources_uiowa.py __________________________________________________________
minst/tests/test_sources_uiowa.py:5: in <module>
    import minst.sources.uiowa as uiowa
minst/sources/__init__.py:1: in <module>
    import philharmonia as philz
E   ImportError: No module named 'philharmonia'
___________________________________________________________ ERROR collecting scripts/tests/test_download.py ____________________________________________________________
scripts/tests/test_download.py:6: in <module>
    import download as DL
E   ImportError: No module named 'download'
================================================================== 25 passed, 4 error in 1.83 seconds ==================================================================
```

**Comment by cjacoby** [comment]
Fixed with #6.

**Comment by ejhumphrey** [comment]
this looks to be resolved.
